tracelib/trace.js

```diff
@@ -11,13 +11,13 @@
   limitations under the License.
 */
 
-var fs = require('fs');
-var mkdirp = require('mkdirp');
-var path = require('path');
-var options = require('./options');
+//var fs = require('fs');
```

Comment:
Intentional comment-out? Maybe add TODO to explain?

Reply:
Fixed this one.